tools/.golangci.yaml

```diff
@@ -40,6 +41,12 @@ linters:
 linters-settings: # please keep this alphabetized
   goimports:
     local-prefixes: go.etcd.io # Put imports beginning with prefix after 3rd-party packages.
+  gomodguard:
+    blocked:
+      modules:
+        - github.com/golang-jwt/jwt/v4:
+            recommendations:
+              - github.com/golang-jwt/jwt/v5
```

Comment:
etcd depends on lots of of 3rd dependencies. I don't see a reason to treat jwt as a special case.
```suggestion
```

Reply:
- Are you going to add an entry here for each dependency when we bump the major version? Why treat jwt as a special case?
- We don't intentionally downgrade any dependencies without a strong justification.